app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :article
+
+  validates :commenter, presence: true
```

Comment:
Wydaje mi się, że tutaj powinien być podany poprawny email.
Może będziemy to walidować?